src/poliastro/atmosphere/coesa62.py

```diff
@@ -149,24 +149,269 @@ def pressure(self, alt, geometric=True):
 
         # Get base parameters
         i = self._get_index(z, self.zb_levels)
+        zb = self.zb_levels[i]
         hb = self.hb_levels[i]
         Tb = self.Tb_levels[i]
         Lb = self.Lb_levels[i]
         pb = self.pb_levels[i]
 
-        # If Z > 90km, different formulas apply
-        if z <= 90 * u.km:
-            print(z, hb, Tb, Lb, pb)
-            if Lb == 0.0:
-                p = pb * np.exp(-g0 * (h - hb) / Tb / R_air)
-            else:
+        # If Lb is zero then apply eqn 1.2.10-(4)
+        if Lb == 0.0:
+            p = pb * np.exp(-g0 * (h - hb) / Tb / R_air)
+        else:
+            # if z <= 90km then apply eqn 1.2.10-(3)
+            if z <= 90 * u.km:
                 T = self.temperature(z)
                 p = pb * (T / Tb) ** (-g0 / R_air / Lb)
-        else:
-            # TODO: Equation (1.2.10) should be applied avobe 90km
-            raise NotImplementedError(
-                "Pressure in COESA62 has just been implemented up to 90km."
-            )
+
+            # if 90 < Z < 700 km then eqn 1.2.10-(5) is applied
+            else:
+                # Converting all the units into SI unit and taking their magnitude
+                Lb_v = Lb.to(u.K / u.m).value
+                r0_v = r0.to(u.m).value
+                z_v = z.to(u.m).value
+                zb_v = zb.to(u.m).value
+                Tb_v = Tb.value
+                g0_v = g0.value
+                R_air_v = R_air.value
+
+                # Putting g = (g0*(r0/(r0 +z))**2) and integrating w.r.t dz in eqn 1.2.10-(5)
```

Comment:
I just left a big comment in the main page for this pull request. Official paper provides an simplification in terms of 8th / 10th degree polynomials for solving the integral part for both pressure and density equations.